resources/migration/resources/site-template/config/engine/site-config.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<site>
+
+    <compatibility>
+        <enableCrafter2ModelConversion>true</enableCrafter2ModelConversion>
```

Comment:
Change the name of the element to: `<enableCrafterModelTypeConversion>`
Add documentation above explaining what this is:
<!-- Starting version 3, Crafter Engine will perform model element type conversion in the Groovy and FreeMarker hosts to help reduce boilerplate work.
Up to and including version 2, Crafter Engine will return model elements as simple strings and the user will have to convert to the correct type.
In version 3 onwards, Crafter Engine will determine the content type based on the type hint and convert the string value to that type before returning it. For example, a model element `start_dt` is determined to be a `Date` type because of the `_dt` hint, and therefore Crafter Engine will return an object of time `Date`. -->

Reply:
Made the changes according to recommendations and discussion from today.